### DISCUSSION TITLE
Text position

### DISCUSSION THREAD

**Discussion by AbisurDiazR** [Lugriz/pdfmake-wrapper#59] [Q&A]
I want to know how set the text position on a pdf page with that dimensions "85x54", I'am trying to add text on pdf page, but only allow me put the text on right side and when i try to center the text, the text is collapsing

**Comment by Lugriz** [comment]
Can you provide a code example about it?

**Comment by AbisurDiazR** [comment]
This my code where i put the text
![image](https://user-images.githubusercontent.com/56562635/151627030-267e0905-162d-4574-b675-ab29d9b7f3d5.png)
The getAddress method return a string with line breaks
![image](https://user-images.githubusercontent.com/56562635/151627121-b4cd0e88-7c4d-4366-8fdd-35e4a0e85eb2.png)
and that is the result
![image](https://user-images.githubusercontent.com/56562635/151627362-acb57ba9-506c-4fc7-9304-3c1d07eb21ec.png)

**Reply by Lugriz** [reply]
What is the expected result? You can use [columns](https://github.com/Lugriz/pdfmake-wrapper#columnscolumns-any---columns) and [stacks](https://github.com/Lugriz/pdfmake-wrapper#stackstack-any---stack) to structure your PDF document, instead of using absolutePosition. Think of columns and stacks to create a GRID.
